Doc/whatsnew/3.11.rst

```diff
@@ -121,7 +121,7 @@ when dealing with deeply nested dictionary objects and multiple function calls,
                                    ~~~~~~~~~~~~~~~~~~^^^^^
     TypeError: 'NoneType' object is not subscriptable
 
-as well as complex arithmetic expressions:
+As well as complex arithmetic expressions:
 
 .. code-block:: python
```

Comment:
```suggestion
As well as complex arithmetic expressions::
```
Same here, and possibly elsewhere.

Reply:
See above—I can do this if you really think its necessary, but I'm not sure it is.